charts/azuremonitor-containers/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 7.0.0-1
 description: Helm chart for deploying Azure Monitor container monitoring agent in Kubernetes
 name: azuremonitor-containers
-version: 2.9.0
+version: 2.9.1
```

Comment:
is this the only place to up the chart version? I saw several other places like deployment scripts ... ?

Reply:
> is this the only place to up the chart version? I saw several other places like deployment scripts ... ?

> Yes, this is only place.